server/models/w-base.js

```diff
@@ -0,0 +1,26 @@
+import { app } from '../server'
+
+module.exports = WBase => {
+  WBase.observe('before save', (ctx, next) => {
+    if (ctx.isNewInstance) {
+      // Add Created timeinfo
+      Object.assign(ctx.instance, {'created_at': new Date()})
+      next()
+    } else if (ctx.instance && ctx.instance.id) {
+      const { WUser } = app.models
+
+      WUser.findById(ctx.instance.id, (err, user) => {
```

Comment:
Yes, can alter the instance directly from the ctx.instance, in the ctx.instance it include the instance that in the request and going to save. 

The `findById` just make a waste on database request for this task. i didn't notice this/